contracts/upgradeable_contracts/arbitrary_message/ForeignAMB.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity 0.4.24;
+
+import "./BasicHomeAMB.sol";
+import "./BasicForeignAMB.sol";
+
+
+contract ForeignAMB is BasicForeignAMB {
+    function initialize() public returns(bool) {
+        require(!isInitialized(), "already initialized");
```

Comment:
Since string revert reasons consume lots of gas during deployment, please avoid to use them. If revert reason is really needed consider to have a uint256 instead.

Reply:
I'll remove them for now, we can add revert reasons as uint256 later if we really need them.